libvast/src/format/mrt.cpp

```diff
@@ -101,14 +133,15 @@ expected<event> reader::read() {
   return no_error;
 }
 
-expected<void> reader::schema(vast::schema const&) {
-  // TODO
-  return make_error(ec::unspecified, "not yet implemented");
+expected<void> reader::schema(vast::schema const& /*sch*/) {
+  return no_error;
```

Comment:
This fails silently, which is harder to debug than the previous version. What's your rationale behind this change?